lms/djangoapps/grades/management/commands/compute_grades.py

```diff
@@ -71,8 +72,28 @@ def add_arguments(self, parser):
             default=0,
             type=int,
         )
+        parser.add_argument(
+            '--estimate_first_attempted',
+            help='Use score data to estimate first_attempted timestamp. Values: (true|false|none)',
+            default='none',
+            choices={'true', 'false', 'none', 'True', 'False', 'None'}
```

Comment:
If the user passes `None`, is that supposed to signify that they want to use the current setting? Would leaving the argument out achieve the same thing?